package.json

```
@@ -15,6 +15,9 @@
   "author": "Formspree, Inc.",
   "license": "MIT",
   "repository": "formspree/netlify-plugin-formspree",
+  "bugs": {
+    "url": "https://help.formspree.io/hc/en-us/requests/new"
```

auto choose correct locale

```suggestion
    "url": "https://help.formspree.io/hc/requests/new"
```